farnaz/challenge5.html

```diff
@@ -0,0 +1,23 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <script>
+        function sortArray(numbers) {
+            //Use the sort() method to sort the array in ascending order
+            numbers.sort(function (a, b) {
```

Comment:
Please consider using another approach instead of the array's `sort` method.

Reply:
var numArray = [140000, 104, 99];
for (var i = 0; i < numArray.length; i++) {
    var target = numArray[i];
    for (var j = i - 1; j >= 0 && (numArray[j] > target); j--) {
        numArray[j+1] = numArray[j];
    }
    numArray[j+1] = target
}
console.log(numArray);

Reply:
Please write this approach in a new commit.